pkg/operator/resource/resourceapply/admissions.go

```diff
@@ -71,7 +72,8 @@ func ApplyMutatingWebhookConfiguration(client admissionclientv1.MutatingWebhookC
 		klog.Infof("ValidatingWebhookConfiguration %q changes: %v", required.Name, JSONPatchNoError(existing, required))
 	}
 
-	actual, err := client.MutatingWebhookConfigurations().Update(context.TODO(), required, metav1.UpdateOptions{})
+	existingCopy.Webhooks = required.Webhooks
```

Comment:
This will clear injected CA bundles. Please check with https://github.com/openshift/library-go/pull/836, which does it IMO a bit better, still, it does not solve Generation tracking properly.

Reply:
I do not think we can make it in applyDirectly func if we use generation to track. And it won't help if we want to change the spec. In this case, should we have a special annotation to hash the applied spec as here https://github.com/openshift/library-go/blob/master/pkg/operator/resource/resourceapply/apps.go#L113-L117?

Reply:
@jsafrane I changed to use generation similar as #836, do you think it makes more sense? I think the generation tracking has to be done on higher layer in this case.